cmd/root.go

```diff
@@ -64,7 +64,7 @@ var RootCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		benchmarkVersion, err := getBenchmarkVersion(kubeVersion, benchmarkVersion, viper.GetViper())
 		if err != nil {
-			exitWithError(err)
+			exitWithError(fmt.Errorf("unable to get benchmark version. error: %v", err))
```

Comment:
NIT: I don't like that we os.Exit in such common code / package. At some point it should be refactored to return wrapped errors so you could properly test it. I know it's not easy though.

For now you could have `exitWithErrorf("unable to get benchmark version: %v", err)`

Reply:
Yeah, we're using that function a lot in **kube-bench**. A bit of refactoring would need to be done. For this PR, I just wanted to add a bit more information to help user troubleshooting.